illumina2vcf/vcf.py

```diff
@@ -218,6 +255,27 @@ def _line_block_to_vcf_line(self, block: List[IlluminaRow], sample_set) -> VCFLi
         for sampleid in sample_set:
             samples.append({"GT": converted_calls.get(sampleid, "./.")})
 
+        # update qc totals if single sample
```

Comment:
🔧 might be useful to put the qc calculation in a separate private function, maybe `_update_qc_from_line(self, vcfline:VCFLine)` ? That will help keep this one from being too long, and make the code more modular and readable, and make it easier to measure the performance penalty that gathering QC applies (could be considerable, for ~700,000 lines)